Purchases/Logging/Strings/IdentityStrings.swift

```diff
@@ -16,19 +16,59 @@ import Foundation
 // swiftlint:disable identifier_name
 enum IdentityStrings {
 
-    static let changing_app_user_id = "Changing App User ID: %@ -> %@"
-    static let logging_in_with_nil_appuserid = "The appUserID passed to logIn is nil or empty. " +
-        "Can't log in. This method should only be called with non-nil and non-empty values."
-    static let logging_in_with_same_appuserid = "The appUserID passed to logIn is the same as the one " +
-        "already cached. No action will be taken."
-    static let creating_alias_success = "Alias created"
-    static let login_success = "Log in successful"
-    static let log_out_called_for_user = "Log out called for user %@"
-    static let log_out_success = "Log out successful"
-    static let creating_alias = "Creating an alias between current appUserID %@ and %@"
-    static let identifying_anon_id = "Identifying from an anonymous ID: %@. An alias will be created."
-    static let identifying_app_user_id = "Identifying App User ID: %@"
-    static let null_currentappuserid = "currentAppUserID is nil. This might happen if the cache in UserDefaults is " +
-        "unintentionally cleared."
+    case changing_app_user_id(from: String, to: String)
+
+    case logging_in_with_nil_appuserid
+
+    case logging_in_with_same_appuserid
+
+    case creating_alias_success
+
+    case login_success
+
+    case log_out_called_for_user(appUserID: String)
+
+    case log_out_success
+
+    case creating_alias(userA: String, userB: String)
```

Comment:
I think this should be `from` and `to` for consistency

Reply:
we could also update the copy

Reply:
I think I like the current copy `between current appUserID \(userA) and \(userB)`, it's not really a from-to relationship right? What were you thinking for copy?

Reply:
you're right, the `from` / `to` situation was different since we were actually switching users, nevermind